react-pallet-uniques/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@mui/material": "^5.4.1",
     "@polkadot/api": "^7.4.1",
     "@polkadot/extension-dapp": "^0.42.6",
```

Comment:
I think we can remove this dependencies, they are already included on [`@subspace/subspace`](https://github.com/subspace/subspace.js/blob/9a6ee5e81b199629e48a49a8230aff3b19ba6f60/package.json#L26-L27)

Reply:
If they are not used explicitly - sure, if they are called directly though, they should stay (I didn't check the code though)

Reply:
we need these, because we have two connections here - one with Subspace (handled by Subspace.js) and another one with Statemine / Statemint (this is where we still need these dependencies)